staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -263,6 +283,19 @@ func (t *tracker) Update(gvr schema.GroupVersionResource, obj runtime.Object, ns
 	return t.add(gvr, obj, ns, true)
 }
 
+func (t *tracker) getWatches(gvr schema.GroupVersionResource, ns string) []*watch.FakeWatcher {
```

Comment:
Yes, thanks.

Reply:
I also added  test for multi-time & multi-thread invocation, WDYT?